qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java

```diff
@@ -312,6 +313,30 @@ private static void testValueResolversFromTemplateEnum(List<ValueResolverInfo> r
 			resolvers);
 	}
 
+	private static void testValueResolversFromTemplateGlobal(List<ValueResolverInfo> resolvers) {
+
+		// @TemplateGlobal
+		// public class Globals {
+
+		// static int age = 40;
+		assertValueResolver(null, "age : int", "org.acme.qute.Globals", resolvers);

```

Comment:
It should be nice to check globalVariable (true).

Your test should look like:

`assertValueResolver(null, "age : int", "org.acme.qute.Globals", true, resolvers);`

Inother words add the globalvariable parameter to assertValueResolver